src/main/java/com/github/alex1304/jdash/client/AbstractGDClient.java

```diff
@@ -414,6 +409,21 @@
 		return fetch(new GDLevelSearchRequest(this, filters, followedAccountIDs, page));
 	}
 
+	/**
+	 * Gets comments of specific level in Geometry dash
+	 *
+	 * @param levelId the level ID to get comments
+	 * @param isRecent Whether to sort in recent order or not(then Most Liked.)
+	 * @param page the page number
+	 * @return a Mono emitting a paginator containing all comments in level. Note that if
+	 *         no comments are found, it will emit an empty paginator. (In this case,
+	 *         fortunately, Geometry Dash API does not return the same response
+	 *         when an actual error occurs while processing the request).
+	 */
+	public Mono<GDPaginator<GDComment>> getLevelCommentById(long levelId, boolean isRecent, int page){
```

Comment:
Two things here:
- The name of the method is kinda misleading, the first argument is the level id not the comment id, and "comment" should take an S since it gets a whole page of comments and not just one. So I suggest a name like `getCommentsForLevel`
- I'd prefer if the `isRecent` parameter is an enum instead of a boolean, even if there are only 2 possibilities it's more intuitive to pass a constant than a boolean.

Reply:
Changed to _getCommentsForLevel_

Changed ```boolean isRecent``` to ```LevelCommentFilter filter```

Reply:
Hmm I don't think it can be called "filter" because filtering and sorting are completely different things. Let's call it `CommentSortMode`.

The rest is good, will probably merge after this last thing is resolved.

Reply:
oof  I guess I still had a lot of problems translating them.
Changed ```LevelCommentFilter filter``` to ```CommentSortMode mode```
Updated _javadoc_ for ```mode``` param